eva_cttv_pipeline/trait_mapping/trait_names_parsing.py

```diff
@@ -28,8 +28,7 @@ def parse_trait_names(filepath: str) -> list:
     nt_expansion_traits = set()
 
     for clinvar_record in clinvar_xml_utils.ClinVarDataset(filepath):
-        trait_names = set(trait.preferred_or_other_name.lower() for trait in clinvar_record.traits
-                          if trait.preferred_or_other_name is not None)
+        trait_names = set(trait.preferred_or_other_valid_name.lower() for trait in clinvar_record.valid_traits)
         for trait_name in trait_names:
             trait_name_counter[trait_name] += 1
```

Comment:
Actually I'm a bit confused about this in general, won't these counts all be 1 since `trait_names` is a set?

Reply:
Aye, that it is, but this set is actually created per `clinvar_record` for each iteration of the cycle. So the same trait name can and will appear multiple times and then be counted in the `trait_name_counter`